src/App.tsx

```diff
@@ -5,7 +5,17 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (newMovie: Movie) => {
+    const repeat = movies.some(movie => movie.imdbId === newMovie.imdbId);
```

Comment:
```suggestion
    const isMoviePresent = movies.some(movie => movie.imdbId === newMovie.imdbId);
```
It would be great to give more meaningful name, so it's clear that it is a boolean value
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig